tools/README.md

```diff
@@ -0,0 +1,15 @@
+# Helpful Tools
+This directory contains a list of tools/scripts that is potentially helpful. 
+
+### Update Old Manifests
+The format of the manifest has been updated in the new developer preview on 11/29/2017. Updates mostly include changes of XML tag names. The details of the updates can be found here {TODO}.
```

Comment:
What is the TODO link going to be? I think this would be a good time for us to flesh out the release notes section of our github: https://tableau.github.io/extensions-api/docs/trex_release-notes.html

@d45 what're your thoughts here?

Reply:
I was hoping to put the new wiki page with the updated tag names once that's don.